src/elements/editor.js

```diff
@@ -340,6 +341,25 @@ export default class LexicalEditorElement extends HTMLElement {
     )
   }
 
+  #handleTab() {
```

Comment:
@zoltanhosszu I think I would move this to the command dispatcher class. We could add a `this.#registerKeyboardCommands` method that registers these. Since this is relative to list formatting, I feel that is a better place, to leave the central editor decoupled for those. I know we are doing the same with ENTER here but ENTER is kind of an special case (not tied to formatting, but we want to intercept and cancel it when pressing CTRL/CMD aloing with it)